### ISSUE TITLE
missing `unstable_feature`

### ISSUE THREAD

**Issue by 0x42h** [draivin/rust-hotswap#5]
```toml
[package]
name = "hotswaptest"
version = "0.1.0"

[lib]
name = "hotswaptest"
crate-type = ["dylib"]
path = "src/main.rs"

[dependencies]
hotswap = "*"
hotswap-runtime = "*"
```

```rs
#![feature(plugin, const_fn)]
#![plugin(hotswap)]
#![hotswap_header]

#[hotswap]
fn sayhello() -> () {
  println!("Hello, world!");
}

fn main() {
   unsafe { hotswap_start!() }
  sayhello();
}
```
```console
~/Projects/Rust/hotswaptest$ cargo run
warning: file found to be present in multiple build targets: /home/bjorn/Projects/Rust/hotswaptest/src/main.rs
   Compiling libc v0.2.40
   Compiling stable_deref_trait v1.0.0
   Compiling libloading v0.4.3
   Compiling smallvec v0.6.0
   Compiling lazy_static v1.0.0
   Compiling hotswap v0.3.1
   Compiling owning_ref v0.3.3
error[E0063]: missing field `unstable_feature` in initializer of `syntax::ext::base::SyntaxExtension`
  --> /home/bjorn/.cargo/registry/src/github.com-1ecc6299db9ec823/hotswap-0.3.1/src/lib.rs:58:9
   |
58 |         NormalTT {
   |         ^^^^^^^^ missing `unstable_feature`

error[E0609]: no field `node` on type `&syntax::ast::Ident`
  --> /home/bjorn/.cargo/registry/src/github.com-1ecc6299db9ec823/hotswap-0.3.1/src/util.rs:44:49
   |
44 |                         ident = Some(span_ident.node);
   |                                                 ^^^^

error: aborting due to 2 previous errors

Some errors occurred: E0063, E0609.
For more information about an error, try `rustc --explain E0063`.
error: Could not compile `hotswap`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```
:broken_heart:

**Comment by draivin** [comment]
Fixed in 0.3.2, commit https://github.com/draivin/rust-hotswap/commit/86a3177e57fb8410f749a5bb46de396080e462d6 ❤️

**Comment by 0x42h** [comment]
```console
hotswaptest$ command cargo run
warning: file found to be present in multiple build targets: hotswaptest/src/main.rs
   Compiling hotswaptest v0.1.0 (hotswaptest)
    Finished dev [unoptimized + debuginfo] target(s) in 5.0 secs
     Running `target/debug/hotswaptest`
Hello hotswap!
Hello hotswap!
Hello hotswap!
Hello hotswap!
Hello hotswap!
^Z
hotswaptest$ vim src/main.rs
hotswaptest$ cargo build --lib && fg
warning: file found to be present in multiple build targets: hotswaptest/src/main.rs
   Compiling hotswaptest v0.1.0 (hotswaptest)
    Finished dev [unoptimized + debuginfo] target(s) in 1.98 secs
cargo run
Hello hotswap!
I LOVE hotswap!
I LOVE hotswap!
I LOVE hotswap!
I LOVE hotswap!
```
:heart_eyes: ... *sigh*

:pray:
